### PR TITLE
Add xfig

### PR DESCRIPTION
**Pull request by epruesse**

### CODE REVIEW

**Comment by epruesse** [comment]
AppVeyor fails to even start due to missing curl (posted as issue #2652)

**Comment by mariusvniekerk** [comment]
Can you rebase this?  Seems to be a lot of commits that aren't relevant in this pull request.

**Comment by epruesse** [comment]
@mariusvniekerk Fixed

**Comment by mariusvniekerk** [comment]
@jakirkham?

File: recipes/xfig/build.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+
+# Point to the right pkgconfig
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig"
+# Add the build-dir's lib dir to the linker flags
+export LDFLAGS="-L$PREFIX/lib"
```
Comment (ericdill) [inline recipes/xfig/build.sh]: I think this to be `export LDFLAGS="${LDFLAGS} -L$PREFIX/lib" so you don't overwrite [what's being set in the toolchain activate.sh](https://github.com/conda-forge/toolchain-feedstock/blob/master/recipe/activate.sh#L34)

File: recipes/xfig/build.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# Add the build-dir's incude dir to the compiler flags
+export CFLAGS="-I$PREFIX/include"
```
Comment (ericdill) [inline recipes/xfig/build.sh]: Same here. I think this to be `export CFLAGS="${CFLAGS} -I$PREFIX/include" so you don't overwrite [what's being set in the toolchain activate.sh](https://github.com/conda-forge/toolchain-feedstock/blob/master/recipe/activate.sh#L34).

File: recipes/xfig/build.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	# compatible malloc).
+	export LDFLAGS="$LDFLAGS -Wl,-rpath,$PREFIX/lib"
+	;;
+esac
+
+./configure --prefix="$PREFIX" --disable-dependency-tracking
+
+make -j$CPU_COUNT
+
+make install
```
Comment (ericdill) [inline recipes/xfig/build.sh]: Is there a `make test` that could be run also?

File: recipes/xfig/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libpng
+    - jpeg
```
Comment (ericdill) [inline recipes/xfig/meta.yaml]: Please pin `jpeg` and `libpng` as defined [here](https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L28-L71) for both build and runtime requirements. Those pinnings are `jpeg 9*` and `libpng >=1.6.28,<1.7`

**Comment by epruesse** [comment]
@ericdill Done. Thx for review!

**Comment by ericdill** [comment]
Appveyor is actually done and ✅ . Not sure why github still thinks its running. Guess the webhook got swallowed by something.  This LGTM. Anything that I missed can be fixed in the feedstock. Thanks @epruesse

**Pull request merged**